editor/src/components/canvas/canvas-strategies/shared-absolute-move-strategy-helpers.ts

```diff
@@ -198,3 +198,10 @@ export function getFileOfElement(
     (_success, _element, _underlyingTarget, underlyingFilePath) => underlyingFilePath,
   )
 }
+
+// No need to include descendants in multiselection when dragging
+export function getDragTargets(selectedViews: Array<ElementPath>): Array<ElementPath> {
+  return selectedViews.filter((view) =>
```

Comment:
maybe worth adding a comment here to note that if you select lots of elements this may be slow